group_vars/all.yaml

```diff
@@ -56,6 +56,14 @@ dask_gateway:
   tcp_port: 8011
   environment: dask-gateway
 
+ipyparallel:
+  enable_mpi: False
```

Comment:
It might be nice to have this be a boolean that becomes true if `enable_bodo: true`

https://stackoverflow.com/questions/54313664/ansible-if-else-with-vars-passing-to-different-playbook

Also, I think `False` shouldn't be capitalized in Ansible land

Reply:
Yeah I'm interested in how to expose this. See https://github.com/Quansight/qhub-onprem/pull/67/files#diff-96a2c3e7b841f4b6f8fcc9fdefdc26e6ebe9147d0b329bc292e4b87184cd75c2R49 where mpi is enabled/disabled. The problem is that the environment must have mpi4py installed in order for the `--mpi` flag to work

Reply:
Might be part of a larger discussion on how to automate the deployment. I don't think it should block this from being merged though.

merging this in